packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -26,17 +38,17 @@ export const UserMessage = ( {
 		shouldUseHelpCenterExperience,
 		trackEvent,
 		chat,
+		userHasEverEscalatedToHumanSupport,
 	} = useOdieAssistantContext();
 
 	const hasCannedResponse = message.context?.flags?.canned_response;
 	const isRequestingHumanSupport = message.context?.flags?.forward_to_human_support;
 	const hasFeedback = !! message?.rating_value;
 	const isBot = message.role === 'bot';
-	const isConnectedToZendesk = chat?.provider === 'zendesk';
 	const isPositiveFeedback =
 		hasFeedback && message && message.rating_value && +message.rating_value === 1;
 	const showExtraContactOptions =
-		( hasFeedback && ! isPositiveFeedback ) || isRequestingHumanSupport;
+		hasFeedback && ! isPositiveFeedback && userHasEverEscalatedToHumanSupport;
```

Comment:
If we follow @sirino suggestion (p1736434881255269), this code would be:

```
const showExtraContactOptions =
userHasEverEscalatedToHumanSupport;
```

Reply:
The current code doesn't make sense: it will only show the contact options if the user rated negatively AND has asked for an escalation.
I prompted:
- human support
- I need help installing plugins
- human support
and got the "It sounds you want to talk to a human. Let me connect you with a Happiness Engineer..." response, but not contact buttons. This doesn't look right.

Reply:
I'm a bit confused here, I can't replicate what you did. Have you proxied and forced version 16?

Reply:
The new changes have fixed this issue, I'll mark this thread as resolved!